packages/plugin-github/src/providers/documentationFiles.ts

```diff
@@ -0,0 +1,51 @@
+import { elizaLogger, IAgentRuntime, Memory, Provider } from "@ai16z/eliza";
+import { GitHubService } from "../services/github";
+
+export const documentationFilesProvider: Provider = {
```

Comment:
Looks like a lot of code duplication can we create a wrapper function that we pass in the files we want for each provider that will drastically clean this up